src/datepicker/datepicker-service.ts

```diff
@@ -151,6 +151,15 @@ export class NgbDatepickerService {
     return this._calendar.isValid(ngbDate) ? ngbDate : defaultValue;
   }
 
+  getMonth(struct: NgbDateStruct) {
```

Comment:
should be `NgbDate`, no ?

Reply:
It's called from NgbDatepickerMonthView with NgbDateStruct argument. So I would like to keep it as it is.